window/src/main/java/controller/WindowController.java

```diff
@@ -1,24 +1,31 @@
 package controller;
 
 import model.WindowModel;
+import view.SwingWindow;
 
 import java.util.concurrent.TimeUnit;
 
 /*
  * control window, created with Swing
- * TODO SwingWindow
+ *
  */
 public class WindowController {
     private final WindowModel WINDOW_MODEL;
-//  GUI;
+    private final SwingWindow GUI;
 
     public WindowController(WindowModel windowModel) {
         WINDOW_MODEL = windowModel;
+        GUI = new SwingWindow();
     }
 
+
     public void start() throws InterruptedException {
+        GUI.startWindow(WINDOW_MODEL, this);
         while (true) {
-//            GUI.repaint();
+            if (!GUI.resized()) {
+                GUI.updateButtonsAndSliders(WINDOW_MODEL);
```

Comment:
jakie slajdery?

Reply:
No na ten moment nie ma żadnych sliderów. Jeżeli będziemy potrzebowali jakiekolwiek to mamy już funkcję za to odpowiedzialną. Jak chcesz to można to usunąć, bo nwm czy będziemy ich nawet używać

Reply:
no usun pozniej dodamy

Reply:
bo tak wprowadza w zaklopotanie